kolibri/core/static/assets/fonts/noto-full.ach-ug.modern.css

```diff
@@ -98,6 +98,24 @@
   font-display: swap;
 }
 
+@font-face {
+  font-family: 'noto-full';
+  src: url('noto-full.NotoSansKhmer.400.woff') format('woff');
+  font-style: normal;
+  font-weight: normal;
+  unicode-range: U+1780-17DD,U+17E0-17E9,U+17F0-17F9,U+19E0-19FF;
+  font-display: swap;
+}
+
+@font-face {
+  font-family: 'noto-full';
+  src: url('noto-full.NotoSansKhmer.700.woff') format('woff');
```

Comment:
nice!

Reply:
Seeing these files always make want to mutter _woof, woof..._! :dog: :joy: